documentation/rdme.md

```diff
@@ -98,162 +109,97 @@ While there are [dozens of event options available](https://docs.github.com/acti
+- [Syncing a directory of Markdown files](https://docs.readme.com/docs/github-actions-docs-example)
 
-In the example above, every push to the `main` branch will check out your repository's contents and sync the contents of the `documentation` directory with your ReadMe project.
+## Usage in Other CI Environments
 
-### Example: Using GitHub Secrets
+Since `rdme` is a command-line tool at its core, you can use `rdme` to sync your documentation from virtually any CI/CD environment that runs shell commands—[Bitbucket Pipelines](https://support.atlassian.com/bitbucket-cloud/docs/get-started-with-bitbucket-pipelines/), [GitLab CI/CD](https://docs.gitlab.com/ee/ci/), you name it! You can do this by invoking `rdme` with `npx rdme@RDME_VERSION` in a Node.js environment. See below for several examples.
 
 > 🚧 Secretly store your ReadMe API Key!
 >
-> GitHub Actions has [secrets](https://docs.github.com/actions/security-guides/encrypted-secrets) to securely store sensitive information so it isn't publicly visible. We **strongly** recommend using these for storing your ReadMe API Key, your API definition ID, and any other secret keys—whether your repository is public or private. You can read more about setting these up [in their documentation](https://docs.github.com/actions/security-guides/encrypted-secrets).
-
-To use a GitHub secret in your `rdme` GitHub Action, first [create a new repository secret](https://docs.github.com/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository). Let's say you create a new secret key called `README_API_KEY`. The usage in the `rdme` step will look something like this:
-
-```yml
-- uses: readmeio/rdme@RDME_VERSION
-  with:
-    rdme: openapi [path-to-file.json] --key=${{ secrets.README_API_KEY }} --id=${{ secrets.README_API_DEFINITION_ID }}
-```
-
-## Usage in Other CI Environments
-
-Since `rdme` is a command-line tool at its core, you can use `rdme` to sync your documentation from virtually any CI/CD environment that runs shell commands—[Bitbucket Pipelines](https://support.atlassian.com/bitbucket-cloud/docs/get-started-with-bitbucket-pipelines/), [GitLab CI/CD](https://docs.gitlab.com/ee/ci/), you name it! You can do this by invoking `rdme` with `npx rdme@RDME_VERSION` in a Node.js environment. See below for several examples.
+> Nearly every CI service has a way to securely add secrets so that they're not exposed in your scripts and build logs. We strongly recommend using such a feature for storing your ReadMe API key. The following examples use `$README_API_KEY`, which is how you typically load such variables in your scripts. We've included some links below on how to configure these for the respective examples:
+>
+> - [Bitbucket Pipelines](https://support.atlassian.com/bitbucket-cloud/docs/variables-and-secrets/#Secured-variables)
+> - [GitLab CI](https://docs.gitlab.com/ee/ci/variables/#add-a-cicd-variable-to-a-project)
 
```

Comment:
You might want to add [CircleCI encrypted environment variables](https://circleci.com/docs/env-vars/) and [Travis CI encrypted files](https://docs.travis-ci.com/user/encrypting-files/) too.

Reply:
Added here: https://github.com/readmeio/rdme/pull/637/commits/2df9a2bd5295aa958c296ccbc86118d58ef71dc3

I used this link for Travis CI instead: https://docs.travis-ci.com/user/environment-variables

Also added some actual example code too, let me know what you think!